### ISSUE TITLE
Link Purpose should be clear

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8129]
### Describe your problem(s)

[2.4.4 Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=141%2C1413%2C211%2C245%2C246%2C247%2C248%2C249%2C253%2C321%2C325%2C331%2C244#link-purpose-in-context)
The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context, except where the purpose of the link would be ambiguous to users in general.

### Describe your solution

1. [G91](https://www.w3.org/WAI/WCAG21/Techniques/general/G91.html) - Add a descriptive text
2. [H30](https://www.w3.org/WAI/WCAG21/Techniques/html/H30.html) - Using the alt attribute for the img element to describe the purpose of a graphical link.
3. [ARIA7](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA7.html) - Use aria-labelledby when using a generic text like Read more.. 
4. [ARIA8](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8.html) - describe the purpose of a link
